static/src/js/components/GranuleResults/GranuleResultsItem.js

```diff
@@ -92,16 +89,16 @@ const GranuleResultsItem = forwardRef(({
 
   const buildThumbnail = () => {
     let element = null
-
     if (granuleThumbnail) {
       element = (
-        // eslint-disable-next-line jsx-a11y/img-redundant-alt
-        <img
+        <EDSCImage
           className="granule-results-item__thumb-image"
           src={granuleThumbnail}
-          height={thumbnailHeight}
-          width={thumbnailWidth}
+          height={85}
```

Comment:
We can change this one to pull from the static.json.config. We can't do it in all places (unless we make a new object to just store the different sizes which I think would actually be more confusing) because in some other spots the h,w are different values. I switched it back in this file to pulling from the `static.config.json`